apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.ts

```diff
@@ -63,15 +66,49 @@ router.get(
 router.get(
   '/',
   asyncHandler(async (req, res) => {
+    const manualGradingPreviewEnabled = req.query.manual_grading_preview === 'true';
+    if (manualGradingPreviewEnabled) {
+      res.locals.manualGradingInterface = true;
```

Comment:
In addition to rendering `pl-manual-grading-only` and not rendering `pl-hide-in-manual-grading`, this will also cause the elements to be disabled (non-editable) and will show the correct answer by default. I believe this is the intended purpose, I just want to make sure this has been considered, and if so I believe a comment in that regard may be useful.

Also, since the elements will be disabled, it probably makes sense to also remove the footer, or at the very least disable the Save/Grade buttons. What do you think?

Reply:
I think these are both reasonable suggestions. By "a comment", do you mean in the code, or something user-facing?

Reply:
I mean in the code.

Reply:
I added a comment in 37960d6. Removal of footer to come...

Reply:
Footer removed in 72494e1 by setting `questionContext: 'manual_grading'` when in the manual grading preview mode.

Reply:
I believe this causes the copy button to no longer show?
https://github.com/PrairieLearn/PrairieLearn/blob/72494e1ae56aee51a413e347ed02656aa8f2652d/apps/prairielearn/src/components/QuestionContainer.html.ts#L646-L651

Reply:
Which, upon testing, seems to not be a big deal since you can still copy in regular view.

Reply:
This actually causes an error though: in regular view, create at least 4 submissions, then in manual grading view try to open the first submission. This will cause an error because of:
https://github.com/PrairieLearn/PrairieLearn/blob/72494e1ae56aee51a413e347ed02656aa8f2652d/apps/prairielearn/src/components/SubmissionPanel.html.ts#L75-L80

Reply:
Oof, excellent catch! How about a dedicated `showFooter` flag for `QuestionContainer` then?

Reply:
That would probably be more robust. Another option is to replace the conditional above with a check if `instance_question` exists (which presumes a question-based URL), but that feels a bit like a bandaid...

Reply:
Agreed, I think explicit flags are going to be more robust. Done in 071c7ea.